readthedocs/builds/tasks.py

```diff
@@ -376,7 +376,7 @@ def send_build_status(build_pk, commit, status, link_to_build=False):
     if provider_name in [GITHUB_BRAND, GITLAB_BRAND]:
         # get the service class for the project e.g: GitHubService.
         service_class = build.project.git_service_class()
-        users = build.project.users.all()
```

Comment:
Can we have validation in the linter that detects the usage of `projects.users.` and complains?

Reply:
that would be great. I don't know if there is a tool that does this.